bin/zkEnv.sh

```diff
@@ -120,6 +120,7 @@ for d in "$ZOOBINDIR"/../zookeeper-server/target/lib/*.jar
 do
    CLASSPATH="$d:$CLASSPATH"
 done
+export CLASSPATH
```

Comment:
Behaviorally, I don't think it matters, but stylistically, it always seemed to me that it would be best to export after making all assignments. There's a few lines below that still assign `CLASSPATH` to another value.

However, I don't think it matters, because I just did a quick test with two bash scripts, and it seems like if you export and then assign it a new value, the exported value is the new value, even if it was assigned after the export statement. I don't know if that works in all cases, though.

In any case, I still think it's good style to export after all assignments, rather than after some and before others.